src/main/resources/application.yml

```diff
@@ -12,11 +12,11 @@ spring:
 chain:
   id: ${IEXEC_CHAIN_ID:134}
   sidechain: ${IEXEC_IS_SIDECHAIN:true}
-  privateAddress: ${IEXEC_PRIVATE_CHAIN_ADDRESS:https://bellecour.iex.ec}
-  hubAddress: ${IEXEC_HUB_ADDRESS:0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f}
-  blockTime: ${IEXEC_BLOCK_TIME:PT5S}
-  gasPriceMultiplier: ${IEXEC_GAS_PRICE_MULTIPLIER:1.0} # txs will be sent with networkGasPrice*gasPriceMultiplier, 4.0 means super fast
-  gasPriceCap: ${IEXEC_GAS_PRICE_CAP:22000000000} #in Wei, will be used for txs if networkGasPrice*gasPriceMultiplier > gasPriceCap
+  node-address: ${IEXEC_PRIVATE_CHAIN_ADDRESS:https://bellecour.iex.ec}
```

Comment:
```suggestion
  node-address: ${IEXEC_BLOCKCHAIN_ADDRESS:https://bellecour.iex.ec}
```

Reply:
We want to stick close to properties in variable.
Renaming this property entails renaming the variable, which will lead us to the breaking change
The README needs to be updated as well